client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/index.tsx

```diff
@@ -43,12 +43,14 @@ export default function SiteContent( {
 		addPageArgs( pageNumber );
 	};
 
+	const showLoader = isFetching && ! sites.length;
```

Comment:
`isFetching` is true if the query is fetching from the API whether or not the data is cached. Because of this if the user has no sites then the loading state will still show. If you check `status === 'loading'` then the loader will only display on the initial fetch of the data, which is what I think we are trying to do here.

Reply:
Yes, good suggestion. We are now using isLoading(status === 'loading') instead of isFetching.